assets/methods_description_template.yml

```diff
@@ -3,11 +3,10 @@ description: "Suggested text and references to use when describing pipeline usag
 section_name: "Arcadia-Science/metagenomics Methods Description"
 section_href: "https://github.com/Arcadia-Science/metagenomics"
 plot_type: "html"
-## TODO nf-core: Update the HTML below to your prefered methods description, e.g. add publication citation for this pipeline
 ## You inject any metadata in the Nextflow '${workflow}' object
 data: |
   <h4>Methods</h4>
-  <p>Data was processed using Arcadia-Science/metagenomics v${workflow.manifest.version} ${doi_text} of the nf-core collection of workflows (<a href="https://doi.org/10.1038/s41587-020-0439-x">Ewels <em>et al.</em>, 2020</a>).</p>
+  <p>Data was processed using Arcadia-Science/metagenomics v${workflow.manifest.version} ${doi_text} using nf-core best practices (<a href="https://doi.org/10.1038/s41587-020-0439-x">Ewels <em>et al.</em>, 2020</a>).</p>
```

Comment:
This reads a little weird as currently written. As a reader of a manuscript methods section, I would be very confused what this meant if i saw this